Merge documentation of installing scala-cli on MacOs and MacOs/M1 (#1507)
VirtusLab/scala-cli: website/docs/_advanced_install.mdx

>
<TabItem value="manual">

> If you are using M1, run following commands:

Reviewer: ```suggestion
When installing Scala CLI on an M1 Mac, run the following commands:
```

> For Mac with **arm64** architecture, run the following commands:

Reviewer: ```suggestion
For a Mac with the **arm64** architecture run the following commands:
```

```bash
curl -fL https://github.com/Virtuslab/scala-cli/releases/latest/download/scala-cli-aarch64-apple-darwin.gz | gzip -d > scala-cli
chmod +x scala-cli
mv scala-cli /usr/local/bin/scala-cli
```

> If you are using different architecture than M1, run following commands:

Reviewer: ```suggestion
When installing Scala CLI on a non-M1 Mac, run the following commands:
```

> For Mac with **x86_64** architecture, run the following commands:

Reviewer: ```suggestion
Otherwise, for a Mac with non-**arm64** architecture (pre-M1) run the following commands:
```

<DownloadButton desc= 'Scala CLI for MacOS' href='https://github.com/Virtuslab/scala-cli/releases/latest/download/scala-cli-x86_64-apple-darwin.pkg'></DownloadButton>
<br/>
<br/>
<DownloadButton desc= 'Scala CLI for MacOS/M1' href='https://github.com/Virtuslab/scala-cli/releases/latest/download/scala-cli-aarch64-apple-darwin.pkg'></DownloadButton>
<br/>
<br/>

> Once downloaded, right-click on downloaded file from Finder and then choose "Open".

Reviewer: ```suggestion
Once downloaded, right-click on the downloaded file from Finder and then choose "Open".
```

This method is provided and supported by the community, not the core team of Scala CLI
:::

:::info

> Please note that arm64 architecture is not yet supported with this method.

Reviewer: ```suggestion
Please note that the `arm64` architecture is not yet supported with this method.
```

:::

> Please note that M1 is not supported.

Reviewer: ```suggestion
:::info
Please note that M1 is not yet supported with this method.
:::
```

> Scala CLI can be installed with [Nix](https://nixos.org) with:

Reviewer: ```suggestion
Scala CLI can be installed via [Nix](https://nixos.org) with:
```
